Makefile

```diff
@@ -8,8 +7,8 @@ TEST_FILES=$(shell find test/*.coffee)
 # for now, exactly the same as .all
 .pretestem: .all
 
-.app: $(APP_FILES)
-	node_modules/.bin/browserify $(APP_FILES) -o dist/twine.js -t coffeeify
+.app:
+	node_modules/.bin/coffee -p lib/assets/javascripts/twine.coffee > dist/twine.js
```

Comment:
@tylerball only way to get rid of the comment.

Seems like all the requests to make that comment optional were declined... https://github.com/jashkenas/coffeescript/pull/3166